### PR TITLE
Software release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: software_versioned_docs/version-0.25/release-notes.md [hunk 1]
```diff
@@ -12,6 +12,20 @@ If you're upgrading to Astronomer v0.30, see [Upgrade to Astronomer v0.25](upgra
 
 We're committed to testing all Astronomer Software versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
 
+## 0.25.14
+
+Release date: September 21, 2021
```
Comment (jwitz) [inline software_versioned_docs/version-0.25/release-notes.md:17]:
```suggestion
Release date: September 21, 2022
```

File: software_versioned_docs/version-0.28/release-notes.md [hunk 2]
```diff
@@ -15,6 +15,27 @@ This document includes all release notes for Astronomer Software v0.28.
...
+
+Release date: September 21, 2022
+
+### Additional improvements
+
+- You can now specify `authUrlParams` for your identity provider (IdP) in `config.yaml`
+- Added support for [Kubernetes 1.22](https://kubernetes.io/blog/2021/08/04/kubernetes-1-22-release-announcement/)
+- Upgraded Prometheus to the LTS release of 2.37.0
+
+### Bug fixes
```
Comment (jwitz) [inline software_versioned_docs/version-0.28/release-notes.md:28]: @cmarteepants @vishwas-astro Did this change make it into 0.28? https://github.com/astronomer/issues/issues/4323

File: software_versioned_docs/version-0.25/release-notes.md [hunk 3]
```diff
@@ -12,6 +12,20 @@ If you're upgrading to Astronomer v0.30, see [Upgrade to Astronomer v0.25](upgra
 
 We're committed to testing all Astronomer Software versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.25/release-notes.md:13]:
```suggestion
All Astronomer Software versions are tested for scale, reliability and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).
```

File: software_versioned_docs/version-0.25/release-notes.md [hunk 4]
```diff
@@ -12,6 +12,20 @@ If you're upgrading to Astronomer v0.30, see [Upgrade to Astronomer v0.25](upgra
...
+
+### Additional improvements
+
+- Improved the startup time for the NATS server
+- Upgraded Prometheus to the LTS release of 2.37.0
+- Removed all remaining uses of Helm2
+
+### Bug fixes
+
+- Fixed several CVEs by upgrading images for system components
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.25/release-notes.md:27]:
```suggestion
- Fixed several Common Vulnerabilities and Exposures (CVEs) by upgrading images for system components
```

File: software_versioned_docs/version-0.28/release-notes.md [hunk 5]
```diff
@@ -15,6 +15,27 @@ This document includes all release notes for Astronomer Software v0.28.
 
 We're committed to testing all Astronomer Software versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.28/release-notes.md:16]:
```suggestion
All Astronomer Software versions are tested for scale, reliability and security on Amazon EKS, Google GKE, and Azure AKS. If you have questions or an issue to report, contact [Astronomer support](https://support.astronomer.io).
```

File: software_versioned_docs/version-0.28/release-notes.md [hunk 6]
```diff
@@ -15,6 +15,27 @@ This document includes all release notes for Astronomer Software v0.28.
...
+
+### Additional improvements
+
+- You can now specify `authUrlParams` for your identity provider (IdP) in `config.yaml`
+- Added support for [Kubernetes 1.22](https://kubernetes.io/blog/2021/08/04/kubernetes-1-22-release-announcement/)
+- Upgraded Prometheus to the LTS release of 2.37.0
+
+### Bug fixes
+
+- Fixed the following CVEs:
```
Comment (kwcanuck) [inline software_versioned_docs/version-0.28/release-notes.md:30]:
```suggestion
- Fixed the following Common Vulnerabilities and Exposures (CVEs):
```

**Review by kwcanuck — APPROVED** [review]

File: software_versioned_docs/version-0.25/release-notes.md [hunk 7]
```diff
@@ -12,6 +12,20 @@ If you're upgrading to Astronomer v0.30, see [Upgrade to Astronomer v0.25](upgra
...
+## 0.25.14
+
+Release date: September 21, 2022
+
+### Additional improvements
+
+- Improved the startup time for the NATS server
+- Upgraded Prometheus to the LTS release of 2.37.0
+- Removed all remaining uses of Helm2
+
```
Comment (vishwas-astro) [inline software_versioned_docs/version-0.25/release-notes.md:24]: @jwitz - what about https://github.com/astronomer/issues/issues/4882? Dont we want an entry for that here?

Comment (jwitz) [reply software_versioned_docs/version-0.25/release-notes.md:24]: I din't see the issue tagged for 0.25, is it in that release?

Comment (vishwas-astro) [reply software_versioned_docs/version-0.25/release-notes.md:24]: my bad .. i thought this was 0.28 release note

**Pull request merged**